README.md

```diff
@@ -33,22 +34,35 @@ Read more about [SUIT CSS](https://github.com/suitcss/suit/).
 * `Grid--alignRight`: right-align all child `Grid-cell`
 * `Grid--alignMiddle`: middle-align all child `Grid-cell`
 * `Grid--alignBottom`: bottom-align all child `Grid-cell`
+* `Grid--fit`: evenly distribute space amongst all child `Grid-cell`
```

Comment:
Yeah, I think with equalHeight existing I wondered if equal alone is confusing. Copied Philip Waltons grid here on the end. Open to suggestions of course